server/handlers/contexts_handler.go

```diff
@@ -82,6 +82,13 @@ func (h *Handler) DeleteContext(w http.ResponseWriter, req *http.Request, prefOb
 		return
 	}
 	h.config.K8scontextChannel.PublishContext()
+
+	err = models.FlushMeshSyncData(req.Context(), mux.Vars(req)["id"], provider)
```

Comment:
Can you make this asynchronous by putting this under a go routine and returning immediately?
Use eventStreamer instance to stream back the result/error of this function to the client.

Reply:
Basically do not return a 500 in case the meshsync data flush fails. If the context was deleted then it is 200. The rest of the information regarding data cleanup can be part of events.

Reply:
makes sense, will make this change.